src/ProgressBar.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bulma;
+
+use Yiisoft\Html\Html;
+
+/**
+ * Progress Bar widget
+ * Native HTML progress bar
+ * 
+ * ```php
+ * echo ProgressBar::widget()->progressValue(75);
+ * ```
+ * 
+ * @link https://bulma.io/documentation/elements/progress/
+ */
+final class ProgressBar extends Widget
+{
+    private array $options = [];
+    private float $progressValue = 0;
+    private ?int $progressMax = 100;
+    private string $size = '';
+    private string $color = '';
+
+    protected function run(): string
+    {
+        $this->buildOptions();
+
+        return $this->renderProgressBar();
+    }
+
+    protected function renderProgressBar(): string
+    {
+        return Html::tag('progress', "{$this->progressValue}%", $this->options);
+    }
+
+    private function buildOptions(): void
+    {
+        if (!isset($this->options['id'])) {
+            $this->options['id'] = "{$this->getId()}-progressbar";
+        }
+
+        $this->options = $this->addOptions($this->options, 'progress');
+
+        if ($this->progressMax !== null) {
+            $this->options['max'] = $this->progressMax;
+        }
+
+        if ($this->progressValue > 0) {
```

Comment:
0 is a valid value and it's different than not specifying a value. 0 is "no progress", nothing is an animation.

Reply:
You're right. thanks!